### PR TITLE
Include JSONFormatter in bundle

### PR DESCRIPTION
**Pull request by mhamann**
- Converted build to use webpack
- Upgraded babel version
- Included `JSONFormatter` in build by default
- Removed bower deps in favor of npm

### CODE REVIEW

**Comment by mohsen1** [comment]
very nice! I'll merge it tonight

**Comment by mhamann** [comment]
@mohsen1 and updates on merging this and publishing a new version to npm?

**Comment by mohsen1** [comment]
Hey @mhamann this is great but we need to make sure it works end-to-end
I couldn't make a build and confirm this is working and don't have enough time to dig into it.

I would like to move away from bower and gulp and instead build it with webpack and test it with Jest

Can you look into it?

Thanks!

**Comment by mhamann** [comment]
@mohsen1 yeah, i can try to find some time to work on that. (cc: @chrisdudley - maybe someone from your team could collab w/ me on this)

I see the PR was reverted. Did something break?

I did run some manual tests to ensure this worked and solved the issue--and it did. But certainly could break people if they were depending on the bower side of things.

This probably deserves a `2.0.0` version bump.

**Comment by mohsen1** [comment]
The bower dependencies were still in the demo app. What I wish I could do is to have webpack dev server for demo and webpack for building the js/css assets

**Comment by mhamann** [comment]
Gotcha. Webpack dev server shouldn't be a problem.

**Comment by mitar** [comment]
I tried the version at this commit (1f87948570c31e4fce0f0086da69099157a6516d) and I can use it in my code with npm + browserify with:

```
import JSONSchemaView from 'json-schema-view-js';
```

On the current master branch (with commit reverted) I cannot because it does not find `./helpers.js` during browserify  bundling.

One thing I had to add was json-formatter-js dependency, which was not automatically installed.

File: package.json [hunk 1]
```diff
@@ -44,6 +48,12 @@
     "karma-mocha": "^0.2.0",
     "karma-mocha-reporter": "^1.1.1",
     "minimist": "^1.2.0",
-    "mocha": "^2.2.5"
+    "mocha": "^2.2.5",
+    "uglifyjs-webpack-plugin": "^1.0.0-beta.3",
+    "webpack": "^3.8.1",
+    "webpack-stream": "^4.0.0"
+  },
+  "dependencies": {
```
Comment (mitar) [inline package.json:56]: This `dependencies` key overrides the `dependencies` key above, which contains `json-formatter-js`. You have a duplicate JSON key here.

**Pull request merged**